# Publish all joint states -----------------------------------
limo_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 100
/gazebo_ros_control/pid_gains:
    front_left_wheel: {p: 1.0, i: 0, d: 0.01} 
    front_right_wheel: {p: 1.0, i: 0, d: 0.01} 
    rear_left_wheel: {p: 1.0, i: 0, d: 0.01} 
    rear_right_wheel: {p: 1.0, i: 0, d: 0.01} 
    
front_left_wheel_velocity_controller:
  type: effort_controllers/JointVelocityController
  joint: front_left_wheel

front_right_wheel_velocity_controller:
  type: effort_controllers/JointVelocityController
  joint: front_right_wheel

rear_left_wheel_velocity_controller:
  type: effort_controllers/JointVelocityController
  joint: rear_left_wheel

rear_right_wheel_velocity_controller:
  type: effort_controllers/JointVelocityController
  joint: rear_right_wheel
